ts/output/chtml/FontData.ts

```diff
@@ -38,6 +38,7 @@ export * from '../common/FontData.js';
 export interface ChtmlCharOptions extends CharOptions {
   c?: string;                   // the content value (for css)
   f?: string;                   // the font postfix (for css)
+  F?: string;                   // the full font css class (for extensions)
```

Comment:
Maybe use a different small letter? I find this confusing.

Reply:
OK, i've changed to `ff` for `full font`.